src/SaveHandler/Cache.php

```diff
@@ -130,32 +117,18 @@ public function gc($maxlifetime)
 
     /**
      * Set cache storage
-     *
-     * @return Cache
      */
-    public function setCacheStorage(CacheStorage $cacheStorage)
+    public function setCacheStorage(CacheStorage $cacheStorage): Cache
```

Comment:
This method should be deprecated and dropped and the constructor changed to use property promotion